client/modules/IDE/components/Editor/oldClassComponent.jsx

```diff
@@ -0,0 +1,716 @@
+// TODO: convert to functional component
```

Comment:
In order to help keep the repository directory easy to navigate, ideally we would not create a new file holding the component, and would just make our changes directly to the `index.jsx` file. 

Because of the way Git works, the history would be preserved in case we would ever want to look back at the old component!